web-console/4.0-designs/forms/forms.md

```diff
@@ -0,0 +1,38 @@
+# Forms
+
+## Labels
+
+* Labels should be semi-bold and written in headline style.
+* Labels should be top-aligned.
+* Required fields should be identified with a red asterisk to the right of the label.
+
+![Labels example](img/service.png)
+
+## Buttons
+
+* Button labels should be written in headline style.
+* In embedded forms and side panel forms, confirmation buttons should be left aligned at the bottom of the page. Cancel buttons should be to the right of confirmation buttons.
+* In modal forms, confirmation buttons should be right aligned at the bottom of the modal. Cancel buttons should be to the left of confirmation buttons.
+* Submit, Save, and other confirmation buttons should always be enabled.
+
+![Buttons example left](img/silence-crop.png)
+
+![Buttons example right](img/project-crop.png)
+
+## Fields
+
+* Input fields and dropdowns in embedded forms should provide enough space for an appropriate input but should never span the full width of a page. The max width is 650px.
```

Comment:
I'd expect that it's alright (and even necessary) to have full width fields on very small viewports

Reply:
Great point! Will reword this 👍